packages/maskbook/src/components/RequestPermission/RequestPermission.tsx

```diff
@@ -37,8 +40,8 @@ export function RequestPermission(props: RequestPermissionProps) {
                 </List>
             </DialogContent>
             <DialogActions>
-                <Button onClick={props.onCancel} variant="text">
```

Comment:
I changed the cancel functionality to a reject functionality. The difference being that we remember whether a user has rejected a permission and won't ask them anymore. But cancel is equivalent to closing the tab.